oshi-core/src/main/java/oshi/software/os/linux/LinuxFileSystem.java

```diff
@@ -65,14 +65,30 @@
 
     @Override
     public List<OSFileStore> getFileStores(boolean localOnly) {
+        //Map of volume with device path as key
+        Map<String, String> volumeDeviceMap = new HashMap<>();
+        File volumes = new File("/dev/mapper");
+        if (volumes.listFiles() != null) {
```

Comment:
Just noticed this on second review.  You call `listFiles()` twice, which is extra I/O.

Reply:
fixed this one and also the below uuid call.